src/Raven.Server/Documents/Indexes/MapReduce/Static/MapReduceIndex.cs

```diff
@@ -277,8 +278,8 @@ public static Index Open<TStaticIndex>(StorageEnvironment environment, DocumentD
 
             return instance;
         }
-
-        public static MapReduceIndex CreateNew<TStaticIndex>(IndexDefinition definition, DocumentDatabase documentDatabase, bool isIndexReset = false)
+        
+        public static MapReduceIndex CreateNew<TStaticIndex>(IndexDefinition definition, DocumentDatabase documentDatabase, bool isIndexReset = false, SingleIndexConfiguration configuration = null)
```

Comment:
I think that renaming `configuration` to `forcedConfiguration` will express the intention of this optional parameter better. What is you opinion?